qemu/tests/cpu_add.py

```diff
@@ -138,8 +140,9 @@ def onoff_para_opt(onoff_params):
     vcpu_been_pluged = 0
     for i in range(vcpu_need_hotplug):
         hotplug_vcpu_params = params.object_params("hotplug_vcpu%s" % i)
-        plug_cpu_id = len(vm.vcpu_threads)
-        plug_cpu_id = hotplug_vcpu_params.get("id", plug_cpu_id)
+        plug_cpu_id = hotplug_vcpu_params.get("id")
```

Comment:
id is "smp_8.8192m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_blk.up.virtio_net.RHEL.7.3.x86_64.io-github-autotest-qemu.cpu_add_test.one_cpu"